dev.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.11-slim-bookworm
+
+WORKDIR /app/simulator_worker
+
+COPY simulator-worker/requirements.txt /app/requirements.txt
+RUN pip install --no-cache-dir -r /app/requirements.txt
+
+COPY ../omotes-sdk-protocol/python/ /omotes-sdk-protocol/python/
+COPY ../omotes-sdk-python/ /omotes-sdk-python/
+RUN pip install -e /omotes-sdk-python/
+RUN pip install -e /omotes-sdk-protocol/python/
+
+COPY simulator-worker/  /app/simulator_worker/
+#COPY simulator-worker/.git /app/simulator_worker/.git
```

Comment:
This gave issues running the tests. With the line commented out it works.  
Is this COPY necessary? It is active in the `Dockerfile`.

Reply:
Not necessary,  its needed if you're installing simulator-worker as a python package,  since setuptools-git-versioning needs a git repo.   

On the other hand, setuptools-git-versioning  is not really needed for simulator-worker, as it's never published as a package.

Reply:
Ahh thanks for clarifying @MichielTukker ! Good one. Then perhaps we just remove setuptools-git-versioning from pyproject.toml?

Reply:
Removed it in dev.Dockerfile.